apollo-router/src/plugins/telemetry/config_new/fixtures/router/custom_histogram_duration_aborted_request_with_condition_on_response/router.yaml

```diff
@@ -0,0 +1,16 @@
+telemetry:
+  instrumentation:
+    instruments:
+      router:
+        http.server.active_requests: false
+        http.server.request.duration: false
+        "custom.histogram.duration":
+          description: "histogram of requests"
+          type: histogram
+          unit: "unit"
+          value: duration
+          # This instrument should not be triggered on drop as the condition is never true
+          condition:
+            eq:
+              - response_header: "never-true"
+              - static: "true"
```

Comment:
Is it intentional to not use `"true"` alone without the `static` field ?

Reply:
I was under the impression we were moving away from specifying values without using static.

Reply:
But here for conditions it's an enum called `SelectorOrValue` and so you can still use a `Value` and not a selector

Reply:
Ahh, good point. I can change them.

Reply:
4d6f58d